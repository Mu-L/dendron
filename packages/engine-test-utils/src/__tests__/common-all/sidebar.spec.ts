describe("GIVEN sidebars object", () => {
  describe("WHEN parsing", () => {
    test.todo("THEN should do something");
  });
});
