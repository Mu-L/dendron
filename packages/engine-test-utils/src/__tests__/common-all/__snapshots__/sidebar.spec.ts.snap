// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GIVEN sidebar config input WHEN containing autogenrated sidebar item THEN return resolved sidebar 1`] = `
Array [
  Object {
    "id": "foo.ch1",
    "label": "Ch1",
    "type": "note",
  },
]
`;

exports[`GIVEN sidebar config input WHEN providing DefaultSidebar THEN return default sidebar 1`] = `
Array [
  Object {
    "id": "bar",
    "label": "Bar",
    "type": "note",
  },
  Object {
    "items": Array [
      Object {
        "id": "foo.ch1",
        "label": "Ch1",
        "type": "note",
      },
    ],
    "label": "Foo",
    "link": Object {
      "id": "foo",
      "type": "note",
    },
    "type": "category",
  },
]
`;
